--- "/home/aditya/Aditya/IIT Bombay/Freshman Year/CS108_Project/CS108_Bash_Grader/.my_git/3467A17390BB7096/quiz2.csv"	2024-05-03 18:51:24.772450139 +0530
+++ "/home/aditya/Aditya/IIT Bombay/Freshman Year/CS108_Project/CS108_Bash_Grader/quiz2.csv"	2024-05-03 18:52:36.657019437 +0530
@@ -1,181 +1,180 @@
 Roll_Number,Name,Marks
-210050102,Murala Vamshi,7.3
-23B0901,Kaushal Vijayvergiya,4.33
-23B0902,Arya Suwalka,3.47
-23B0903,Abhi Jain,4.89
-23B0904,Prakhar Jain,4.33
-23B0905,Sagnik Nandi,5.61
-23B0906,Velagala Deeraj Sathvik Reddy,9.8
-23B0907,Hari Shankar Karthik,2.82
-23B0908,Raghav Goyal,8.15
-23B0909,Varri Shyam Sri Vardhan,9.05
-23B0910,Sonal Kumari,7.36
-23B0911,Kalla Lakshmi Surya,5.75
-23B0912,Sanapala Venkata Aditya,2.87
-23B0913,Dipen Sojitra,0
-23B0914,Suvvada Mounisha Naidu,4.72
-23B0915,Nandipati Sai Durga Reddy,6.83
-23B0916,Vanguri Eshwar,5.7
-23B0917,Rajat Das,7.38
-23B0918,Banoth Tharun Tej,9.63
-23B0919,Yougank Kumar,3.02
-23B0920,Maradana Dheeraj Kumar,5.55
-23B0921,Ishita,7.28
-23B0922,Akshara,6.96
-23B0923,Abhineet Majety,4.88
-23B0924,Bondala Somanath Vasanth Yadav,7.63
-23B0925,Vangapandu Venkata Sai Surya Prakash,6.33
-23B0926,Chaitanyaa Mahesh Maheshwari,1.24
-23B0927,Gowri K Balakrishnan,6.3
-23B0928,Anushka Singhal,5.87
-23B0929,Nayakanti Naga Bhavya Sree,7.76
-23B0930,Mehul Borad,3.56
-23B0931,Haresh Gupta,4.81
-23B0932,Tejas Narendra Chaudhari,9.37
-23B0933,Sarthak Dhananjay Thakare,6.55
-23B0934,Yash Singh,7.51
-23B0935,Potnuru Saatvika,6.85
-23B0936,Meesala Pranathi Sreeja,3.1
-23B0937,Nikhil Narera,5.18
-23B0938,Deshank Pratap Singh,9.06
-23B0939,Bathula Rithvik,3.41
-23B0940,Aditya Neeraje,5.16
-23B0941,Maradana Anusri,9.48
-23B0942,Dhruv Sanjay Jain,6.95
-23B0943,Shaik Sahil Chanda,4.56
-23B0945,Akshit Kumar,5.91
-23B0947,Akepati Sriya,8.26
-23B0948,Kritin Gupta,6.64
-23B0949,Kamyak Channa,6.55
-23B0950,Harshit Raj,8.15
-23B0951,Jatsya Jariwala,2.45
-23B0952,Punumalli Lohit Adhitya Sai,9.31
-23B0953,Aakash Gupta,6.17
-23B0954,Bavurupudi Ritwik,10
-23B0955,Gulshan Kumar,6.67
-23B0956,Siddhant Mulkikar,6.57
-23B0957,Kunumalla Gautam Siddharth,4.81
-23B0958,Satyam Sinoliya,6.04
-23B0960,J Adarsh,7.81
-23B0961,Arijit Mandal,3.1
-23B0962,Banothu Sonusree,3.52
-23B0963,Komrelly Snigdha Reddy,3.57
-23B0964,Seemala Varsha,5.01
-23B0965,Nakka Mahathi,4.48
-23B0966,Ananya Latchupatula,4.86
-23B0967,Guduru Adarsh,7.28
-23B0968,Suthar Harshul Sanjaybhai,4.46
-23B0969,Yakkanti Pani Venkata Maneendhar Reddy,4.92
-23B0970,Priyanshu Kumar,7.22
-23B0971,Mayank Soni,4.43
-23B0972,Komarapu Vivek Vardhan,7.32
-23B0973,Sammeta Avaneesh Sai,7.46
-23B0974,Krish Gupta,6.05
-23B0975,Menda V Venkata Harsha Vardhan,2.89
-23B0976,Boyina Sanjana,6.62
-23B0977,Keyaan Kartikeya Rajesh,4.02
-23B0978,Gowdu Charishma,4.56
-23B0979,Satyala Madhu Kiran,7.08
-23B0980,Mradul Sonkar,10
-23B0981,Repudi Akshay Karthik,9.2
-23B0982,Navya Garg,6.39
-23B0983,Konda Puspa Kumar Reddy,8.77
-23B0984,Varala Charishma,5.06
-23B0985,Jashanpreet Singh,4.0
-23B0986,Tanishq Mahesh Mandhane,6.59
-23B0987,Dasari Saketh Naidu,7.12
-23B0988,Prabhav Khandelwal,8.82
-23B0989,Gaurav Kumar,4.62
-23B0990,Pothina Madhav,3.06
-23B0991,Shravya Karna,2.63
-23B0992,Dumpala Paneendra Nadha Reddy,4.86
-23B0993,Kethavath Nithin Naik,5.11
-23B0994,Vavilala Chidvilas Reddy,7.65
-23B0995,Kunj Bhesaniya,6.54
-23B0996,Punit Kumar,1.39
-23B0997,Thammaneni Dheeraj,6.03
-23B0998,Velamakanni Satwik,7.91
-23B0999,Yuvraj Gupta,5.45
-23B1000,Paluri Gnana Kousik Reddy,4.83
-23B1002,Sasthrula Joel Tenny,8.54
-23B1003,Abhinav Singh,2.14
-23B1004,Karri Ganga Dhana Sri,2.96
-23B1005,Sara Mukherjee,7.59
-23B1006,Sandeep Reddy Nallamilli,5.26
-23B1007,Shivam Singh Yadav,2.16
-23B1008,Jainendrajeet,7.83
-23B1009,Udit Sarkar,7.39
-23B1010,Pudi Ajay Kumar,10
-23B1011,Abhilasha Sharma Suman,7.51
-23B1012,Burri Gehna Chelvi,6.48
-23B1014,Peela Teja Sri,5.88
-23B1015,Pinnu Shasank Reddy,9.67
-23B1016,Harshil Amitkumar Solanki,1.28
-23B1017,Evuri Mohana Sreedhara Reddy,5.81
-23B1018,Bikkina Abhinav Chowdary,7.12
-23B1020,Chennu Rakesh Kumar Reddy,10
-23B1021,Sambangi Varshini,7.54
-23B1022,Rayapati Dinesh Sai,3.51
-23B1023,Pushpendra Uikey,5.04
-23B1024,Pendurthi Nischal Subhash,7.92
-23B1025,Somya Dahiaya,3.69
-23B1026,Duggineni Venkata Yugesh,6.89
-23B1027,Dola Surya Sai,5.52
-23B1028,Ashish Kumar,7.1
-23B1029,Karedla Balaji,2.53
-23B1030,Vaka Sri Varshitha,9.57
-23B1031,Mrunal Shrikant Vairagade,7.75
-23B1032,Vysyaraju Srikar Nihal,6.35
-23B1033,Chinthaparthi Babu Sujan Reddy,10
-23B1034,Malay Kedia,1.92
-23B1035,Sameer Arvind Patil,5.23
-23B1036,Kabir S Prakash,7.72
-23B1037,Vennela Yasasvi Naidu,5.16
-23B1038,N.K.Vishwaajith,5.54
-23B1039,Nenavath Sree Vamshi Madhav,7.46
-23B1040,B Shashank,5.36
-23B1041,Merchant Dhruvraj Vijaykumar,5.84
-23B1042,Sangadi Vijaya Raghavendra,6.26
-23B1043,Yash Vikas Sabale,6.51
-23B1044,Moulik Jindal,6.47
-23B1045,Niral Charan,9.56
-23B1046,Kokkiligadda Manaswi,6.0
-23B1047,Tapase Aditi Maniklal,5.37
-23B1048,Sanchita Chaurasia,1.62
-23B1049,Ridam Jain,8.95
-23B1050,Ujwal L Shankar,6.01
-23B1051,Aman Nehra,6.3
-23B1052,Ramachandran S,7.26
-23B1053,Aditi Singh,7.78
-23B1054,Pathi Nikitha,6.35
-23B1055,Sanapala Anirudh,9.18
-23B1056,Addagada Venkata Sivaram,6.83
-23B1057,Sabil Ahmad,9.79
-23B1058,Udtyavath Sai Likhith,5.12
-23B1059,Allam Sujay,7.12
-23B1060,Madhav R Babu,8.77
-23B1061,Devangam Kishan Teja,5.22
-23B1062,Ramesh Surya Theja,4.26
-23B1063,Gunda Sushanth,4.99
-23B1064,Chaudhari Avinash Janardhan,5.92
-23B1065,Devang Tibrewal,6.66
-23B1066,Kunsoth Chandrashen Nayak,4.87
-23B1067,Harsh Suthar,2.07
-23B1068,Vaibhav Singh,10
-23B1069,Uday Gajanan Darade,8.26
-23B1071,Inampudi Harshitha,7.07
-23B1072,Ette Hasini,4.1
-23B1073,Marumamula Venkata Pranay,3.7
-23B1074,Saksham Jain,6.38
-23B1075,Shiv Narang,5.15
-23B1076,Aryan Mahajan,7.83
-23B1077,Bhavya Bansal,4.47
-23B1078,Saksham Saharia,4.78
-23B1079,Sumedh S S,6.78
-23B1080,Shaik Awez Mehtab,3.56
-23B1081,Rishi Kalra,5.28
-23B1082,Dheeravath Thanuj,1.62
-23B1083,Jigyasa Chouhan,8.07
-23B1084,Guthikonda Abhiram,3.81
-23B1085,Harith S,3.54
-23V0066,Hsi Yu Ho,6.05
+210050102,Murala Vamshi,8.27
+23B0901,Kaushal Vijayvergiya,3.82
+23B0902,Arya Suwalka,5.75
+23B0903,Abhi Jain,7.23
+23B0904,Prakhar Jain,6.19
+23B0905,Sagnik Nandi,5.15
+23B0906,Velagala Deeraj Sathvik Reddy,6.01
+23B0907,Hari Shankar Karthik,7.98
+23B0908,Raghav Goyal,5.14
+23B0909,Varri Shyam Sri Vardhan,6.69
+23B0910,Sonal Kumari,8.36
+23B0912,Sanapala Venkata Aditya,5.03
+23B0913,Dipen Sojitra,4.72
+23B0914,Suvvada Mounisha Naidu,5.99
+23B0915,Nandipati Sai Durga Reddy,6.35
+23B0916,Vanguri Eshwar,8.54
+23B0917,Rajat Das,6.38
+23B0918,Banoth Tharun Tej,5.03
+23B0919,Yougank Kumar,5.09
+23B0920,Maradana Dheeraj Kumar,5.07
+23B0921,Ishita,6.73
+23B0922,Akshara,8.98
+23B0923,Abhineet Majety,4.85
+23B0924,Bondala Somanath Vasanth Yadav,2.67
+23B0925,Vangapandu Venkata Sai Surya Prakash,3.7
+23B0926,Chaitanyaa Mahesh Maheshwari,5.73
+23B0927,Gowri K Balakrishnan,5.11
+23B0928,Anushka Singhal,5.65
+23B0929,Nayakanti Naga Bhavya Sree,7.43
+23B0930,Mehul Borad,7.35
+23B0931,Haresh Gupta,4.6
+23B0932,Tejas Narendra Chaudhari,4.59
+23B0933,Sarthak Dhananjay Thakare,5.38
+23B0934,Yash Singh,8.03
+23B0935,Potnuru Saatvika,4.06
+23B0936,Meesala Pranathi Sreeja,5.64
+23B0938,Deshank Pratap Singh,6.28
+23B0939,Bathula Rithvik,9.02
+23B0940,Aditya Neeraje,6.61
+23B0941,Maradana Anusri,9.1
+23B0942,Dhruv Sanjay Jain,6.93
+23B0943,Shaik Sahil Chanda,8.41
+23B0944,Aayush Martand Borkar,1.3
+23B0945,Akshit Kumar,6.38
+23B0946,Itthagoni Sathwik Goud,6.77
+23B0947,Akepati Sriya,5.52
+23B0948,Kritin Gupta,10
+23B0949,Kamyak Channa,7.21
+23B0950,Harshit Raj,6.15
+23B0951,Jatsya Jariwala,8.61
+23B0952,Punumalli Lohit Adhitya Sai,3.38
+23B0953,Aakash Gupta,6.55
+23B0954,Bavurupudi Ritwik,4.63
+23B0955,Gulshan Kumar,4.0
+23B0957,Kunumalla Gautam Siddharth,3.96
+23B0958,Satyam Sinoliya,6.14
+23B0960,J Adarsh,5.15
+23B0961,Arijit Mandal,5.82
+23B0962,Banothu Sonusree,5.04
+23B0963,Komrelly Snigdha Reddy,6.91
+23B0964,Seemala Varsha,7.25
+23B0965,Nakka Mahathi,9.44
+23B0966,Ananya Latchupatula,4.56
+23B0967,Guduru Adarsh,7.48
+23B0969,Yakkanti Pani Venkata Maneendhar Reddy,8.01
+23B0970,Priyanshu Kumar,4.0
+23B0971,Mayank Soni,4.7
+23B0972,Komarapu Vivek Vardhan,6.2
+23B0973,Sammeta Avaneesh Sai,5.57
+23B0974,Krish Gupta,0.63
+23B0975,Menda V Venkata Harsha Vardhan,2.52
+23B0976,Boyina Sanjana,6.59
+23B0977,Keyaan Kartikeya Rajesh,0.73
+23B0978,Gowdu Charishma,1.83
+23B0979,Satyala Madhu Kiran,7.5
+23B0981,Repudi Akshay Karthik,5.1
+23B0982,Navya Garg,4.49
+23B0983,Konda Puspa Kumar Reddy,3.96
+23B0984,Varala Charishma,6.27
+23B0985,Jashanpreet Singh,9.55
+23B0986,Tanishq Mahesh Mandhane,5.91
+23B0987,Dasari Saketh Naidu,7.89
+23B0988,Prabhav Khandelwal,2.22
+23B0989,Gaurav Kumar,6.69
+23B0990,Pothina Madhav,6.95
+23B0991,Shravya Karna,7.94
+23B0992,Dumpala Paneendra Nadha Reddy,7.1
+23B0993,Kethavath Nithin Naik,10
+23B0994,Vavilala Chidvilas Reddy,5.83
+23B0995,Kunj Bhesaniya,8.02
+23B0996,Punit Kumar,4.93
+23B0997,Thammaneni Dheeraj,4.32
+23B0998,Velamakanni Satwik,7.84
+23B0999,Yuvraj Gupta,8.65
+23B1000,Paluri Gnana Kousik Reddy,6.02
+23B1001,Vadde Shanvitha Reddy,4.84
+23B1002,Sasthrula Joel Tenny,7.14
+23B1003,Abhinav Singh,5.12
+23B1004,Karri Ganga Dhana Sri,8.67
+23B1005,Sara Mukherjee,8.28
+23B1006,Sandeep Reddy Nallamilli,5.95
+23B1007,Shivam Singh Yadav,9.76
+23B1008,Jainendrajeet,4.1
+23B1010,Pudi Ajay Kumar,6.69
+23B1011,Abhilasha Sharma Suman,7.67
+23B1012,Burri Gehna Chelvi,4.39
+23B1014,Peela Teja Sri,7.54
+23B1015,Pinnu Shasank Reddy,5.59
+23B1016,Harshil Amitkumar Solanki,5.3
+23B1017,Evuri Mohana Sreedhara Reddy,3.3
+23B1018,Bikkina Abhinav Chowdary,7.15
+23B1019,Chirag Gautam,4.59
+23B1020,Chennu Rakesh Kumar Reddy,3.98
+23B1021,Sambangi Varshini,5.39
+23B1022,Rayapati Dinesh Sai,4.55
+23B1023,Pushpendra Uikey,7.6
+23B1024,Pendurthi Nischal Subhash,4.7
+23B1025,Somya Dahiaya,6.95
+23B1026,Duggineni Venkata Yugesh,7.46
+23B1027,Dola Surya Sai,6.38
+23B1028,Ashish Kumar,5.74
+23B1029,Karedla Balaji,4.48
+23B1030,Vaka Sri Varshitha,8.14
+23B1031,Mrunal Shrikant Vairagade,6.07
+23B1032,Vysyaraju Srikar Nihal,3.42
+23B1033,Chinthaparthi Babu Sujan Reddy,8.46
+23B1034,Malay Kedia,3.98
+23B1035,Sameer Arvind Patil,6.98
+23B1036,Kabir S Prakash,7.95
+23B1037,Vennela Yasasvi Naidu,1.31
+23B1038,N.K.Vishwaajith,6.38
+23B1039,Nenavath Sree Vamshi Madhav,2.99
+23B1040,B Shashank,3.97
+23B1041,Merchant Dhruvraj Vijaykumar,4.91
+23B1042,Sangadi Vijaya Raghavendra,10
+23B1043,Yash Vikas Sabale,6.98
+23B1044,Moulik Jindal,2.29
+23B1045,Niral Charan,8.14
+23B1046,Kokkiligadda Manaswi,8.82
+23B1047,Tapase Aditi Maniklal,5.61
+23B1048,Sanchita Chaurasia,10
+23B1049,Ridam Jain,3.85
+23B1050,Ujwal L Shankar,8.7
+23B1051,Aman Nehra,9.52
+23B1052,Ramachandran S,6.89
+23B1053,Aditi Singh,5.78
+23B1054,Pathi Nikitha,4.54
+23B1055,Sanapala Anirudh,5.89
+23B1056,Addagada Venkata Sivaram,7.89
+23B1057,Sabil Ahmad,10
+23B1058,Udtyavath Sai Likhith,4.45
+23B1059,Allam Sujay,7.59
+23B1060,Madhav R Babu,8.82
+23B1061,Devangam Kishan Teja,7.09
+23B1062,Ramesh Surya Theja,3.78
+23B1063,Gunda Sushanth,7.43
+23B1064,Chaudhari Avinash Janardhan,3.26
+23B1065,Devang Tibrewal,5.58
+23B1066,Kunsoth Chandrashen Nayak,2.9
+23B1067,Harsh Suthar,7.9
+23B1068,Vaibhav Singh,5.75
+23B1069,Uday Gajanan Darade,3.08
+23B1070,Bhukya Akkshitha Rathod,6.06
+23B1071,Inampudi Harshitha,6.28
+23B1072,Ette Hasini,7.38
+23B1073,Marumamula Venkata Pranay,5.08
+23B1074,Saksham Jain,7.57
+23B1075,Shiv Narang,3.42
+23B1076,Aryan Mahajan,5.43
+23B1077,Bhavya Bansal,7.98
+23B1078,Saksham Saharia,4.63
+23B1079,Sumedh S S,10
+23B1080,Shaik Awez Mehtab,3.75
+23B1081,Rishi Kalra,3.17
+23B1082,Dheeravath Thanuj,5.06
+23B1083,Jigyasa Chouhan,3.15
+23B1084,Guthikonda Abhiram,6.61
+23B1085,Harith S,2.34
+23V0066,Hsi Yu Ho,7.71
