--- "/home/aditya/Aditya/IIT Bombay/Freshman Year/CS108_Project/CS108_Bash_Grader/.my_git/86F6A78AB821A627/quiz1.csv"	2024-05-03 18:51:24.772450139 +0530
+++ "/home/aditya/Aditya/IIT Bombay/Freshman Year/CS108_Project/CS108_Bash_Grader/quiz1.csv"	2024-05-03 18:51:35.293636266 +0530
@@ -37,7 +37,7 @@
 23B0937,Nikhil Narera,20.21
 23B0938,Deshank Pratap Singh,16.34
 23B0939,Bathula Rithvik,23.33
-23B0940,Aditya Neeraje,8.57
+23B0940,Aditya Neeraje,20
 23B0941,Maradana Anusri,16.43
 23B0942,Dhruv Sanjay Jain,17.9
 23B0943,Shaik Sahil Chanda,17.36
